src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import clsx from "clsx";
+import { makeStyles } from "@material-ui/core/styles";
+import Drawer from "@material-ui/core/Drawer";
+import Button from "@material-ui/core/Button";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import MenuIcon from "@material-ui/icons/Menu";
+import { useMediaQuery } from "react-responsive";
+import { Link } from "react-router-dom";
+
+const links = [
+  {
+    name: "Information",
+    endpoint: "/",
+  },
+  {
+    name: "Profile",
+    endpoint: "/profile",
+  },
+  {
+    name: "FAQ",
+    endpoint: "/faq",
+  },
+  {
+    name: "Registration",
+    endpoint: "/registration",
+  },
+];
+
+const useStyles = makeStyles({
+  list: {
+    width: 250,
+  },
+  fullList: {
+    width: "auto",
+  },
+});
+
+const TemporaryDrawer = () => {
```

Comment:
🔧 Rename this component to match the filename.

Reply:
Component renamed